shared/domains/chelonia/chelonia.js

```diff
@@ -224,7 +269,7 @@ sbp('sbp/selectors/register', {
       // but after it's finished. This is used in tandem with
       // queuing the 'chelonia/private/in/handleEvent' selector, defined below.
       // This prevents handleEvent getting called with the wrong previousHEAD for an event.
-      return sbp('okTurtles.eventQueue/queueEvent', contractID, [
+      return sbp('okTurtles.eventQueue/queueEvent', `chelonia/${contractID}`, [
```

Comment:
Could we please keep what was here originally? It's simpler/cleaner and avoids some CPU cycles too.

Reply:
Additionally, now some other code added in a recent PR for login sync depends on it using the original queue based on the name of the contractID.

Reply:
OK, I am not seeing still why this change is needed. As far as I can tell it doesn't actually materially change the behavior of the code, but does manage to break the sideEffect for `'gi.contracts/group/removeMember'`.

So that's a big problem. If you think this change is needed, please give a clear user story of the scenario that breaks without it.